### ISSUE TITLE
Fresh and successful install of OneMore 5.7.0, but add-in won't load

### ISSUE THREAD

**Issue by rubcap** [stevencohn/OneMore#903]
# Problem to Solve

Installer finishes successfully, but add-in won't load in OneNote '2016' Desktop.

# To Reproduce

1. Close OneNote
2. Run the _OneMore_5.7.0_Setupx64.msi_ installer from admin account.
3. Installer finishes successfully. No apparent problems in log file.
4. Open OneNote

# Expected Behavior

The add-in buttons are visible on the Start ribbon and the add-in can be found under _File > Options > Add-ins_

# Screenshots

(none)

# Environment (if applicable)

OneNote '2016' Desktop (Windows 10, 64 bit, OneNote version **2208**, part of Microsoft 365 for Business).

# Log file (OneMoreSetup.log)

          ```starting action: --install .. 13/02/2023 22:23:21
          Installer architecture (x64), OS architecture (x64), requesting (x64)
          OneMore installer running as user NT AUTHORITY\SYSTEM elevated
          
          Register... version 5.7.0
          
          ShutdownOneNoteAction.Install ---
          ONENOTE process not found
          
          ProtocolHandlerAction.Install ---
          step 0: opening HKLM:\Software\Classes\onemore
          step 1: creating HKLM:\Software\Classes\onemore
          step 2: setting properties of HKLM:\Software\Classes\onemore
          step 3: opening HKLM:\Software\Classes\onemore\shell\open\command
          step 4: creating HKLM:\Software\Classes\onemore\shell\open\command
          step 5: setting properties of HKLM:\Software\Classes\onemore\shell\open\command
          verified: key created HKEY_LOCAL_MACHINE\Software\Classes\onemore\shell\open\command
          
          TrustedProtocolAction.Install ---
          translating user XXXXXXXXX-\admin to SID
          registering trusted protocol for user XXXXXXXXX-\admin (S-1-21-3045428-1604505149-63266779-102)
          defaulting Office version to 16
          step 6: opening HKCU:\Software\Policies\Microsoft\Office\16.0\Common\Security\Trusted Protocols\All Applications\onemore:
          step 7: creating HKCU:\Software\Policies\Microsoft\Office\16.0\Common\Security\Trusted Protocols\All Applications\onemore:
          key created HKEY_USERS\S-1-21-30442628-1604505149-63660779-102\Software\Policies\Microsoft\Office\16.0\Common\Security\Trusted Protocols\All Applications\onemore:
          
          EdgeWebViewAction.Install ---
          WebView2 Runtime is already installed
          install completed successfully
          
          --------------------------------------------------
          reentry action: --install-registrywow .. 13/02/2023 22:23:22
          OneMore installer running as user NT AUTHORITY\SYSTEM elevated
          
          RegistryWowAction.Install --- x64:True
          1st chance, could not read CLSID or key missing
          2nd chance, read CLSID ({DC6E480-C3B-49F8-8232-600C20568E})
          read path from LocalServer32 (C:\Program Files\Microsoft Office\Root\Office16\ONENOTE.EXE)
          detected 64-bit install
          WOW cloning not required
          
          --------------------------------------------------
          reentry action: --install-activesetup .. 13/02/2023 22:23:22
          OneMore installer running as user NT AUTHORITY\SYSTEM elevated
          
          ActiveSetupAction.Install ---
          replacing '5.7.0' with '5,7,0'```

# Additional Context

Same install on my work laptop gave no problems at all. Same MSI using admin rights. Add-in loaded from the first time without problems. Only difference I could find: on my work laptop I have OneNote version **2202** installed.

Thank you for this wonderful open-source add-in. Discovered it today. I hope I can make it work on my private laptop as well.

**Comment by stevencohn** [comment]
These types of problems are impossible to diagnose without hands-on assessment. I've heard of it before but have never been able to reproduce it.

**Comment by julie777** [comment]
It would be useful to see all the registry entries that OneMore creates during install

**Comment by stevencohn** [comment]
The installer validates each registry setting as its applied, so unless the validation logic is wrong (which is unlikely since it works for 99% of people) then it may not be that useful. Regardless, I may add it to the new installer. Not this one.

There is also an [iq.ps1](https://github.com/stevencohn/OneMore/blob/main/iq.ps1) script (Installation Qualification) that you can run to validate your install. More of a diagnostic tool for me but if you want to download it and give it a try, that would be great. Run it with the PowerShell command

`PS> .\iq.ps1 -verbose`

**Comment by rubcap** [comment]
Hi, in the meantime I upgraded my laptop to Windows 11. I had some problems with the taskbar on my Win10 install, so I decided to give it a go. This evening, I ran the _OneMore_5.7.1_Setupx64.msi_ installer twice, first to remove the OneMore installation, then to install it (this didn't work the last time I tried). I opened OneNote 2016 desktop and the buttons were there 🤩!

I don't really feel comfortable running the script you mentioned, I hope you don't mind. It wouldn't be useful anyways now that the install was successful, I guess?  Thank you for this wonderful plug-in. I'll recommend it to colleagues and friends!
